src/cljs/org/broadinstitute/firecloud_ui/common/gcs_file_preview.cljs

```diff
@@ -18,12 +17,13 @@
       {:header "File Details"
        :content
        (let [{:keys [data error status]} (:response @state)
-             data-size (when data (data "size"))
+             data-size (:size data)
```

Comment:
I'm not familiar enough with this yet, but is this safe to do if `data` is falsy? The previous code seemed to be trying to guard against that.